interoperator/config/crds/resource_v1alpha1_sfcluster.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: sfclusters.resource.servicefabrik.io
+spec:
+  group: resource.servicefabrik.io
+  names:
+    kind: SFCluster
+    plural: sfclusters
+  scope: Namespaced
```

Comment:
Any reason for `Namespaced` scope? `kubefed` has [`Cluster` scope](https://github.com/kubernetes-sigs/kubefed/blob/master/charts/kubefed/templates/crds.yaml#L14).

Reply:
No they have it [Namespace Scoper](https://github.com/kubernetes-sigs/kubefed/blob/master/charts/kubefed/charts/controllermanager/templates/crds.yaml#L368)

Reply:
My bad. Checked the wrong CRD 🤦‍♂